.github/workflows/kind.yml

```diff
@@ -15,20 +15,13 @@ defaults:
     shell: bash
 
 jobs:
-  kind-tests:
+  quick-checks:
+    name: Quick Checks
```

Comment:
I'm not sure how I feel about "Quick Checks" since it seems too casual, though I don't have great alternatives since it is kind of a catch-all group.

Brainstorming (which seem about equivalent now that I'm trying to come up with some 😆 ):
- Lint, Unit, Deploy Tests
- Preflight Tests

Reply:
I like the sound of Preflight Tests (even though technically it's not "pre" the other tests). Since it is basically a catch-all, I don't want it to be too specifically named, and have to be renamed to stay accurate.